assembl/tests/fixtures/bright_mirror.py

```diff
@@ -117,6 +117,45 @@ def fin():
     return p
 
 
+@pytest.fixture(scope="function")
+def participant_published_post_with_parent_post_for_bright_mirror(
+        request, test_session, discussion, admin_user, participant1_user,
+        bright_mirror, post_published_for_bright_mirror):
+    from assembl.models import Post, Idea, LangString, IdeaRelatedPostLink, PublicationStates
+    from graphene.relay import Node
+    idea_id = bright_mirror
+    raw_id = int(Node.from_global_id(idea_id)[1])
+    idea = Idea.get(raw_id)
+    parent_post_id = post_published_for_bright_mirror.id
+    post = Post(
+        discussion=discussion, creator=participant1_user,
+        subject=LangString.create(u"Published by participant"),
+        body=LangString.create(u"A simple published fiction by participant"),
+        type='post', publication_state=PublicationStates.PUBLISHED,
```

Comment:
we need an Enum for the types of posts

Reply:
Good remark @amensouissi but this modification should be brought in a ticket on its own because can these modification can insert potential regression.